R/calculateOverlap.R

```diff
@@ -82,27 +82,25 @@ setMethod("calculateOverlap", signature = c(x = "SummarizedExperiment"),
         }
         detection <- as.numeric(detection)
         ########################### INPUT CHECK END ############################
-        # Get assay
-        assay <- assay(x, assay.type)
-        
+
         # All the sample pairs
         sample_pairs <- as.matrix(expand.grid(colnames(x), colnames(x)))
-        
+
         # Loop through all sample pairs
         result <- apply(sample_pairs, 1, FUN = function(sample_pair){
           # Get samples
-          sample1 <- assay[ , sample_pair[1]]
-          sample2 <- assay[ , sample_pair[2]]
```

Comment:
You could just use variable name "assay" so it does not look that everything has changed